analysis-environments.html

```diff
@@ -0,0 +1,202 @@
+---
+layout: idr
+title: Analysis environments
+redirect_from:
+  - analysis-environments/
+---
+
+{% assign studies=site.data.notebooks_and_apps.itemListElement | sort: 'name' | where_exp:"item", "item.name contains 'study'" %}
+
+{% assign items=site.data.notebooks_and_apps.itemListElement | sort: 'name' %}
+
+        <!-- begin marketing header -->
+        <header class="marketing-hero">
+            <div class="row homepage text-center">
+
+                <div class="medium-12 columns">
+                    <h1 class="hero-main-header">IDR and Analysis</h1>
+                    <p class="hero-subheader small-10 medium-10 large-10 small-offset-1 medium-offset-1 large-offset-1">
+                     <p>The Image Data Resource (IDR) uses Analysis Environments provided and maintained by third parties.
+                     </p>
+                </div>
+            </div>
+        </header>
+
+        <!-- begin Background section -->
+        <hr class="whitespace">
+        <div class="row column text-center">
+            <h2>Environments</h2>
+        </div>
+        <hr>
+        <div class="row">
+            <div class="small-10 small-centered medium-10 medium-centered columns">
+                <div class="row horizontal">
+                    <div>
+                        <p>
+                        We have a set of notebooks and apps that can be run in several
```

Comment:
In the build version on https://snoopycrimecop.github.io/idr.openmicroscopy.org/linked-environments.html#page-top the word Notebooks is written with a capital N. Not sure how and why.